src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameService.cs

```diff
@@ -99,14 +141,17 @@ internal InlineRenameSession ActiveSession
             {
                 var previousSession = _activeRenameSession;
                 _activeRenameSession = value;
-                ActiveSessionChanged?.Invoke(this, new ActiveSessionChangedEventArgs(previousSession));
+                if (previousSession != null)
+                {
+                    ActiveSessionChanged?.Invoke(this, new ActiveSessionChangedEventArgs(previousSession));
+                }
```

Comment:
why this cahnge?

Reply:
While adding nullable annotations throughout the file, a warning appeared on that line saying that ```previousSession``` could be null (which I believe may be possible?), so I added the check.

Reply:
Let's not change this but right here.  I'm not sure if the implications of what might happen if we no longer fire these events.  

You can use `!` to suppress the nullable warning here. 

Thanks!

Reply:
It looks like that fixed a large amount of the test failures. Thanks for the catch.